apps/admin-x-activitypub/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tryghost/admin-x-activitypub",
-  "version": "0.6.29",
+  "version": "0.6.33",
```

Comment:
Why are we jumping 4 versions?

Reply:
No that's because my editor does this automatically when I use vs code for commits. Will change before merging

Reply:
oh, the pre-commit hook 😅